files/docker/grest/config/postgres/crontab

```diff
@@ -0,0 +1,28 @@
+# Unlike any other crontab you don't have to run the `crontab'
+# command to install the new version when you edit this file
+# and files in /etc/cron.d. These files also have username fields,
+# that none of the other crontabs do.
+
+SHELL=/bin/bash
+PATH=/nix/var/nix/profiles/per-user/guild/profile/bin:/nix/var/nix/profiles/per-user/guild/profile/sbin:/opt/cardano/cnode/scripts:/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/sbin:/usr/local/bin \
+
+# Example of job definition:
+# .---------------- minute (0 - 59)
+# |  .------------- hour (0 - 23)
+# |  |  .---------- day of month (1 - 31)
+# |  |  |  .------- month (1 - 12) OR jan,feb,mar,apr ...
+# |  |  |  |  .---- day of week (0 - 6) (Sunday=0 or 7) OR sun,mon,tue,wed,thu,fri,sat
+# |  |  |  |  |
+# *  *  *  *  * user-name command to be executed
+#17 *    * * *   root    cd / && run-parts --report /etc/cron.hourly
+#25 6    * * *   root    test -x /usr/sbin/anacron || ( cd / && run-parts --report /etc/cron.daily )
+#47 6    * * 7   root    test -x /usr/sbin/anacron || ( cd / && run-parts --report /etc/cron.weekly )
+#52 6    1 * *   root    test -x /usr/sbin/anacron || ( cd / && run-parts --report /etc/cron.monthly )
+
+# Examples
+#*/10 * * * * if [ -z $NOIP2 ]; then /home/guild/.scripts/ip2loc.sh 2>/dev/null; fi
+#*/5 * * * * if [ -z $NOBLW ]; then /home/guild/.scripts/block_watcher.sh 2>/dev/null; fi
+#*/130 * * * * if [ -z $KEEP ]; then find /opt/cardano/cnode/logs/  -name "*.json" -type f -mtime +0 -type f -delete; fi
+#0 */3 * * * if [ -z $NOREC ]; then cp -rf /opt/cardano/cnode/db/* /opt/cardano/cnode/priv/$NETWORK-db/ 2>/dev/null; fi
+*/35 * * * * history -c
```

Comment:
Should this be commented out as well?

Reply:
not influent, history -c cleans just the bash history